shippable.yml

```diff
@@ -3,7 +3,7 @@ env:
     global:
         - COMPOSER_ALLOW_SUPERUSER=1
         - XDMOD_REALMS='jobs,storage'
-        - XDMOD_BRANCH=xdmod10.5
+        - XDMOD_BRANCH=xdmod11.0
         - XDMOD_SOURCE_DIR=$SHIPPABLE_BUILD_DIR/../xdmod
         - XDMOD_INSTALL_DIR=/xdmod
         - XDMOD_MODULE_DIR=$SHIPPABLE_BUILD_DIR
```

Comment:
This file is an artifact of a previous CI system used, so I think this file can be removed altogether.

Reply:
Thanks, I have removed it.